src/table/components/TableHeadWrapper.jsx

```diff
@@ -21,13 +22,19 @@ function TableHeadWrapper({
   children,
 }) {
   const { columns, paginationNeeded } = tableData;
+  const setHeadRowHeight = useContextSelector(TableContext, (value) => value.setHeadRowHeight);
   const isFocusInHead = useContextSelector(TableContext, (value) => value.focusedCellCoord[0] === 0);
   const setFocusedCellCoord = useContextSelector(TableContext, (value) => value.setFocusedCellCoord);
   const headerStyle = useMemo(() => getHeaderStyle(layout, theme), [layout, theme]);
+  const headRowRef = useRef();
+
+  useEffect(() => {
+    setHeadRowHeight(headRowRef.current.clientHeight);
+  }, [headRowRef.current]);
```

Comment:
will this run when you change the header font size?

Reply:
yep, I tested, I can double-check. [headRowRef] does not help rerun.

Reply:
only run when changing the header font size in properties, not in property-panel... I gonna look into it